olive/model/handler/base.py

```diff
@@ -49,6 +50,7 @@ def __init__(
             if isinstance(io_config, (IoConfig, dict))
             else io_config
         )
+        self.generative = generative
```

Comment:
I think we will have to add this to the `to_json` method too https://github.com/microsoft/Olive/blob/13a0faff67e9f3a1054aa822d815989dfceda3d7/olive/model/handler/mixin/json.py#L35